shell/common/pointer_data_dispatcher.h

```diff
@@ -0,0 +1,71 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef POINTER_DATA_DISPATCHER_H_
+#define POINTER_DATA_DISPATCHER_H_
+
+#include "flutter/runtime/runtime_controller.h"
+#include "flutter/shell/common/animator.h"
+
+namespace flutter {
+
+class PointerDataDispatcher {
```

Comment:
Please document this classes in the doxygen format. See [shell.h](https://github.com/flutter/engine/blob/a353f93d370dded7bc7756082fbab8b519679d4d/shell/common/shell.h#L87) for an example. Please be sure to document the threading restrictions and lifecycle of these objects.